docs/03_tutorials/index.md

```diff
@@ -1 +1,2 @@
 1. [BIOS Boot Sequence](bios-boot-sequence.md): Demonstrates how to bootstrap a new EOSIO blockchain
+2. [Tic Tac Toe Game Contract](tic-tac-toe-game-contract.md): Build a sample PvP game contract to apply the EOSIO knowledge you acquired eariler
```

Comment:
I think using Player vs Player instead of PvP would make it more clear

Reply:
For a person who do not know what is PvP the term Player vs Player hardly make any difference. 

On second thought, I will just remove this term the Tic Tac Toe should be a more commonly known term

Reply:
I disagree `Player vs Player game` it is quite self explanatory as opposed to `PvP game`.

Reply:
As a general rule of thumb, when abbreviations/acronyms are to be used in a sentence, it is enclosed in brackets after spelling out the full terms for first instances. Examples:

- AWS Certificate Manager (ACM)
- AWS Identity and Access Management (IAM)
- Elastic Compute Cloud (EC2)

And so on. 

So we may use this:

Build a sample `Player vs Player (PvP)` game contract to apply the EOSIO knowledge you acquired earlier.

Reply:
> I disagree, `Player vs Player game` it is quite self explanatory as opposed to `PvP game`.

We don't have to argue over this detail. If it is some inaccurate or code is not working, let me know. Otherwise, this is good for me